telemetry/harvester_test.go

```diff
@@ -203,7 +203,15 @@ func TestVetCommonAttributes(t *testing.T) {
 			}
 		},
 	)
-	if len(savedErrors) != 3 {
+	if len(savedErrors) != 1 {
+		t.Fatalf("Expected one error but there was %v", len(savedErrors))
+	}
+	errVal, ok := savedErrors[0]["err"]
+	if !ok {
+		t.Fatalf("Missing expected err key.")
+	}
+	errors := strings.Split(errVal.(string), ",")
+	if len(errors) != 3 {
 		t.Fatal(savedErrors)
 	}
```

Comment:
Changing the contract of `vetCommonAttributes` does make testing a bit harder, since in the new code I concatenate any validation errors into a single error string. So on the test side, I have to split the error string back up again.